src/structures/scoreboard.ts

```diff
@@ -296,12 +289,18 @@ export default class Scoreboard {
     }
 
     /**
-     * @param players - user IDs mapped to Players
-     * @returns players IDs, sorted by rank in the game
+     * @returns a mapping of player scores to ranking
      */
-    static getRanking(players: { [userID: string]: Player }): Array<string> {
-        return Object.entries(players)
-            .sort((a, b) => b[1].getScore() - a[1].getScore())
-            .map((x) => x[0]);
+    getRanking(): { [score: number]: number } {
+        const rankingToScore = {};
+        const sortedUniqueScores = [
+            ...new Set(Object.values(this.players).map((x) => x.getScore())),
+        ].sort((a, b) => b - a);
```

Comment:
Not sure if the sort is needed here anymore

Reply:
Yes required